pkg/controllers/user/cis/register.go

```diff
@@ -3,61 +3,84 @@ package cis
 import (
 	"context"
 
-	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
-
 	"github.com/rancher/rancher/pkg/namespace"
 	"github.com/rancher/rancher/pkg/systemaccount"
+	v3 "github.com/rancher/types/apis/management.cattle.io/v3"
 	"github.com/rancher/types/config"
 	"github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // Register initializes the controllers and registers
 func Register(ctx context.Context, userContext *config.UserContext) {
-	logrus.Infof("Registering CIS controller")
-
 	clusterName := userContext.ClusterName
-	clusterLister := userContext.Management.Management.Clusters(metav1.NamespaceAll).Controller().Lister()
+	clusterClient := userContext.Management.Management.Clusters(metav1.NamespaceAll)
+	cluster, err := clusterClient.Get(clusterName, metav1.GetOptions{})
+	if err != nil {
+		logrus.Errorf("error fetching cluster: %v", err)
```

Comment:
👀 👀 do we need retries? 
seeing this approach for the first time, afraid what if miss out on registering the controller totally

Reply:
will add retries